modules/claims_api/spec/requests/v1/disability_compensation_request_spec.rb

```diff
@@ -1129,20 +1129,6 @@ def obj.class
           Timecop.return
```

Comment:
@acovrig 
We're definitely gonna need to add some specs to this file to ensure that the new REGEX behaves as we expect.

We need to test stuff like ::
- make sure that 2023-01-01 is accepted
- make sure that 2023-01-01T01:01:01-5:00 is accepted
- make sure that hello world is not accepted
- make sure that _ (or empty/blank string) is not accepted
- make sure that 2023-01-01T01:01:01-CST is not accepted

@hinzed1127 can you think of any other cases we should test for?

Reply:
that covers everything that immediately comes to mind to me